tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "allowJs": true,
     "watch": true,
     "alwaysStrict": true,
-    "lib": ["es6", "dom"]
+    "lib": ["es2017", "dom"]
```

Comment:
why the change here?

Reply:
`Object.values()` .. so certainly not necessary, but don't see any reason to avoid it. 

More here: https://github.com/Microsoft/TypeScript/issues/10768

Reply:
Hmmm.. but that is failing the node 6 build, so maybe there's one.

Reply:
nice, can't wait for that to land

Reply:
`Object.entries()`?